src/Microsoft.Data.Analysis/RowCursor.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Microsoft.Data.Analysis
+{
+    internal sealed class RowCursor : DataViewRowCursor
```

Comment:
Why did we move this class outside of `DataFrame.IDataView.cs`?

Reply:
Moving this to a new file was part of a refactor I did internally that didn't pan out. I didn't see a reason to move it back, so I left it here. I don't really have a preference, I can move it back if necessary.

Reply:
It makes reviewing the code harder (because not only was it moved, but also changed in the same PR), and causes a property to go from `private` to `internal`.

Reply:
Good enough for me. Moved back.